### ISSUE TITLE
Update to version 2.0.1

### ISSUE THREAD

**Issue by Epoche-Napoleon** [web-vision/wv_deepltranslate#48]
Hello,

after updating from version 2.0.0 to version 2.0.1 using Composer, I get the following error message:

`Too few arguments to function WebVision\WvDeepltranslate\Override\DatabaseRecordList::makeLocalizationPanel(), 2 passed in XXX/typo3/sysext/recordlist/Classes/RecordList/DatabaseRecordList.php on line 13`42 and exactly 3 expected

Running Analyze Database Structure did nothing.

Best wishes
EPOCHE NAPOLEON

**Comment by tpinne** [comment]
I have the same problem. The problem is a difference between TYPO3 v10 and v11. In v10 the function `makeLocalizationPanel` only accepts 2 parameters. In v11 however it accepts 3 parameters. The update from 2.0.1 seems only to be coded/tested against v11.

For now I fix the version to 2.0.0 until this is compatible with v10 again.

**Comment by sypets** [comment]
There is a discrepancy between what's in the README and in the composer.json:

README:

> [Requirements](https://github.com/web-vision/wv_deepltranslate#extension-configuartion)
> TYPO3 8.5 to 8.7.99 (V1.0.0 - 1.0.1)
> TYPO3 9.5.1 to 10.4.99 (V1.0.2 onwards)
> TYPO3 11.5.1 to 11.5.99 (V2.0.1 onwards)

composer.json
```

 "typo3/cms-core": "^9.5 || ^10.4 || ^11.5",
    "typo3/cms-backend": "^9.5 || ^10.4 || ^11.5",
    "typo3/cms-extbase": "^9.5 || ^10.4 || ^11.5",
    "typo3/cms-fluid": "^9.5 || ^10.4 || ^11.5"
```

ext_emconf.php

`  'typo3' => '9.5.1-11.5.99',`

**Comment by tgaertner** [comment]
So v2.0 was  intended to drop compatibility with TYPO3 9/10...

**Comment by christianbltr** [comment]
I can also confirm this issue with version 2.0.1 in TYPO3 10.

**Comment by baschny** [comment]
It would be a pity that due to one interface change the 2.x branch is now made incompatible with TYPO3 v10. Indeed there is a mismatch between what the README states and what composer states

I would suggest to try to make sure 2.x still works with TYPO3 v10 as you gain a bigger user-base for the latest branch - else "forks" will appear just for "compatibility" reasons.

**Comment by NarkNiro** [comment]
The incompatibility in #51 has been fixed with Versino 2.1.0.
